lib/data/projects_manager.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+
+import 'package:protofast/models/project.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class ProjectsManager {
```

Comment:
Did I get it correctly, this class is intended to do all Project Entity (read/write) related work like a Repository? Is it a used conventions in Dart, or can we rename it to something like ProjectRepository to make it more clear? 
Are we gonna have FeatureManager for features in the future?